cf-console-action-bar-button-sample/src/aem-cf-console-admin-1/web-src/src/components/DisplayMetadataModal.js

```diff
@@ -0,0 +1,221 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React, { useState, useEffect } from 'react'
+import ErrorBoundary from 'react-error-boundary'
+import { attach } from "@adobe/uix-guest"
+import {
+  Grid,
+  Flex,
+  Text,
+  ButtonGroup,
+  Button,
+  Heading,
+  TableView,
+  TableHeader,
+  TableBody,
+  Row,
+  Cell,
+  Column,
+  View
+} from '@adobe/react-spectrum'
+import {Accordion, AccordionItem} from '@react/react-spectrum/Accordion';
+import Spinner from './Spinner';
+import { useParams } from "react-router-dom"
+
+import { extensionId } from "./Constants"
+
+import { getContentFragmentsInfo } from '../utils';
+
+const versionColumns = [
+  {name: 'Version Number', uid: 'title'},
+  {name: 'Version ID', uid: 'id'},
+  {name: 'Created At', uid: 'created'},
+  {name: 'Created By', uid: 'createdBy'}
+];
+
+const fieldColumns = [
+  {name: 'Name', uid: 'name'},
+  {name: 'Type', uid: 'type'},
+];
+
+export default function DisplayMetadataModal () {
+  // Fields
+  const [isContentLoading, setContentLoading] = useState(true);
+  const [guestConnection, setGuestConnection] = useState();
+  const [error, setError] = useState(null);
+  const [fragments, setFragments] = useState([]);
+  const { selection } = useParams();
+  const fragmentIds = selection?.split('|') || [];
+
+  console.log('Selected Fragment Ids', fragmentIds);
+  if (!fragmentIds || fragmentIds.length === 0) {
+    console.error('No Content Fragments are selected.');
+    return;
+  }
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const guestConnection = await attach({ id: extensionId })
+
+        setGuestConnection(guestConnection);
+        console.log(guestConnection);
+
+        const auth = guestConnection.sharedContext.get('auth');
+        const aemHost = guestConnection.sharedContext.get('aemHost');
+
+        const fragments = await getContentFragmentsInfo(aemHost, fragmentIds, auth);
+        console.log(fragments);
+        setFragments(fragments);
+      } catch(e) {
+        setError('Unable to retrieve information about selected fragments.');
+      } finally {
+        setContentLoading(false);
+      }
+    })()
+  }, [])
+
+  const onCloseHandler = () => {
+    guestConnection.host.modal.close()
+  }
+
+  return (isContentLoading || !guestConnection) ? (
```

Comment:
Looks like this file needs to be formatted

Reply:
@lenaorobei can you elaborate on this?